primitives/sr-io/src/lib.rs

```diff
@@ -213,6 +213,18 @@ pub trait Storage {
 	fn changes_root(&mut self, parent_hash: &[u8]) -> Option<Vec<u8>> {
 		self.storage_changes_root(parent_hash).ok().and_then(|h| h)
 	}
+
+	/// Get the next key in storage after the given one in lexicographic order.
+	fn next_key(&mut self, key: &[u8]) -> Option<Vec<u8>> {
+		self.next_storage_key(&key)
+	}
+
+	/// Get the next key in storage after the given one in lexicographic order in child storage.
+	fn child_next_key(&mut self, child_storage_key: &[u8], key: &[u8]) -> Option<Vec<u8>> {
+		let storage_key = child_storage_key_or_panic(child_storage_key);
+		self.next_child_storage_key(storage_key, key)
+	}
+	//TODo todo make test for childs
```

Comment:
TODO

Reply:
oh yes I forget to remove it, indeed all implementation of externalities and backend is now tested similarly for next_key and next_child_key